app/controllers/bookings_controller.rb

```diff
@@ -24,17 +30,35 @@ def edit
   # POST /bookings
   # POST /bookings.json
   def create
-    @booking = Booking.new(booking_params)
-
-    respond_to do |format|
+    # begin
+    new
+    @booking.customer_id = session[:customer]
```

Comment:
@mayur-kambariya @AnkurVyas-BTC This is inappropriate way of assigning values to model attributes. this code can be optimized.